src/containers/DocumentForm.js

```diff
@@ -1,334 +1,339 @@
-import React from 'react';
```

Comment:
Please remove the modifications to DocumentForm.js from this PR, as it's not related...